### DISCUSSION TITLE
Updated translation.xlsx file

### DISCUSSION THREAD

**Discussion by efi99** [PredatH0r/ChanSort#252] [Show and tell]
Hi!
I translated the new strings into hungarian. (v2021-07-26)
[Translation2.xlsx](https://github.com/PredatH0r/ChanSort/files/6900960/Translation2.xlsx)

**Comment by PredatH0r** [comment]
Thanks a lot!
Your changes are now included in the source code, the "translation.xlsx" is now also updated (with a few additional changes).
Your translation will be part of the next release.
